src/StorageSync/StorageSync/help/New-AzStorageSyncService.md

```diff
@@ -0,0 +1,122 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/new-Azstoragesyncservice
+schema: 2.0.0
+---
+
+# New-AzStorageSyncService
+
+## SYNOPSIS
+This command will use to create storage sync service.
+
+## SYNTAX
+
+```
+New-AzStorageSyncService [-ResourceGroupName] <String> [-Name] <String> [-Location] <String>
+ [[-Tag] <Hashtable>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
```

Comment:
Tag should not be positional

Reply:
Please rephrase the comment and suggest change

---
In reply to: [255696343](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255696343) [](ancestors = 255696343)